samples/analyze.js

```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
   // [END video_speech_transcription_gcs]
 }
 
+async function analyzeTextGCS(gcsUri) {
+  //[START video_detect_text_gcs]
+  // Imports the Google Cloud Video Intelligence library
+  const Video = require('@google-cloud/video-intelligence').v1;
```

Comment:
Is there a reason we need to specify the `v1` API here?  You should be able to just require without that.

Reply:
This has been fixed.